library/objective-c/EnvoyEngine.h

```diff
@@ -68,14 +68,22 @@ extern const int kEnvoyFilterHeadersStatusStopAllIterationAndBuffer;
 
 @interface EnvoyHTTPFilter : NSObject
 
-@property (nonatomic, strong) NSString *name;
-
 @property (nonatomic, strong) NSArray * (^onRequestHeaders)(EnvoyHeaders *headers, BOOL endStream);
 
 @property (nonatomic, strong) NSArray * (^onResponseHeaders)(EnvoyHeaders *headers, BOOL endStream);
 
 @end
 
+#pragma mark - EnvoyHTTPFilterFactory
+
+@interface EnvoyHTTPFilterFactory : NSObject
+
+@property (nonatomic, strong) NSString *filterName;
+
+@property (nonatomic, strong) EnvoyHTTPFilter * (^create)();
```

Comment:
should we use similar verbs between here and the bridge (init)?

Reply:
I presented the interface to the bridge layer as something like a class/instance, so `init` made sense. I implemented it at this layer as a split between a factory and an implementation, so `create` made sense.

At least, to me.